hedera-mirror-rest/transactions.js

```diff
@@ -567,86 +524,191 @@ const getTransactionsInnerQuery = function (
         Transaction.CONSENSUS_TIMESTAMP
       )}, ctl.consensus_timestamp) = ttl.consensus_timestamp
       order by consensus_timestamp ${order}
-      ${namedLimitQuery}`;
+      ${namedLimitQuery} )
+    select consensus_timestamp,payer_account_id from timestampFilter`;
   }
 
   return transactionOnlyQuery;
 };
 
-const reqToSql = function (req) {
+/**
+ * Transactions queries are organized as follows: First there's a query that selects the timestamps and payer IDs
+ * for the required number of unique transactions. And then queries other tables to extract all relevant
+ * information for those transactions.
+ * This function returns the second query based on the consensus_timestamps list returned by the first query.
+ * Also see: getTransactionTimestampsQuery function
+ *
+ * @param {{Object}[]} timestampQueryRows the results of the relevant timestamps query, which is rows
+ * consisting of consensus_timestamp and payer_account_id attributes. At least one row is required.
+ * @param {String} order Sorting order
+ * @param {Number} limit Max number of rows
+ * @return {Promise} Promise returning the transaction summary results for the provided transaction timestamp rows
+ */
+const getTransactionsSummary = async (timestampQueryRows, order, limit) => {
+  // Nothing with which to query
+  if (_.isNil(timestampQueryRows) || timestampQueryRows.length === 0) {
```

Comment:
This should work to simplify the undefined and empty check:
```suggestion
  if (_.isEmpty(timestampQueryRows)) {
```

Reply:
Thanks! That willl be in the next push.

Reply:
Done